pygeoapi/api.py

```diff
@@ -1419,6 +1421,19 @@ def get_collection_items(
         else:
             skip_geometry = False
 
+        LOGGER.debug('processing cql parameter')
+        cql = request.params.get('cql')
```

Comment:
See the comment above about the correct parameter. Also, another parameter (`filter-lang`) needs to be captured in the query string object. The PostgreSQL provider configuration should advertise which dialects are supported (`CQL-JSON`/`CQL-TEXT` or both)

Reply:
Added by @ximenesuk: https://github.com/geopython/pygeoapi/pull/964/commits/4133de67bd8f0e5de5a7a5f4f008f47ebcd0b163